v0/ai-logic/main.py

```
@@ -1 +1,172 @@
+        return OrchestrationResponse(
+            job_id=node_response["jobId"],
+            provider=routing_decision.provider,
+            mode=routing_decision.mode,
+            routing_reason=routing_decision.reason,
+            estimated_duration=node_response.get("estimatedDuration"),
+            node_api_response=node_response
+        )
+        
+    except Exception as e:
+        logger.error(f"Orchestration failed: {str(e)}")
+        raise HTTPException(status_code=500, detail=f"Orchestration failed: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)